src/App.css

```diff
@@ -0,0 +1,126 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
+    sans-serif;
```

Comment:
- [ ] Kindly follow the design guidelines as much as possible. You're free to personalize the content on the design but are asked to use exact font-face, weight, and size as per the design guidelines. For this design, the original design guidelines tell that they used [Lato font](https://fonts.google.com/specimen/Lato?query=Lato). Please import this font from the google font that is in the link above and make use of it. 👍